qa/tasks/cephfs/test_subvolume.py

```diff
@@ -132,6 +132,36 @@ def test_subvolume_hardlink_across_subvolumes(self):
             self.mount_a.run_shell(['ln', 'group/subvol2/file1',
                                     'group/subvol3/file1'])
 
+    def test_subvolume_setfattr_empty_value(self):
+        """
+        To verify that an empty value fails on subvolume xattr
+        """
+
+        # create subvol
+        self.mount_a.run_shell(['mkdir', 'group/subvol4'])
+
+        try:
+            self.mount_a.run_shell(['setfattr', '-n', 'ceph.dir.subvolume',
+                                    'group/subvol4'])
+        except CommandFailedError:
+            pass
+        else:
+            self.fail("run_shell should raise CommandFailedError")
+
+    def test_subvolume_rmattr(self):
+        """
+        To verify that rmattr can be used to reset subvolume xattr
+        """
+
+        # create subvol
+        self.mount_a.run_shell(['mkdir', 'group/subvol4'])
+        self.mount_a.run_shell(['setfattr', '-n', 'ceph.dir.subvolume',
+                                '-v', '1', 'group/subvol4'])
+
+        # clear subvolume flag
+        self.mount_a.run_shell(['setfattr', '-x', 'ceph.dir.subvolume',
+                                'group/subvol4'])
```

Comment:
Please use `self.mount_a.setfattr` and create a new `self.mount_a.removexattr`. This is to facilitate compatibility with other OS like Darwin. (@leonid-s-usov is working on this.)

Reply:
Done. Do we have plans to refactor existing `run_shell(['setfattr'...`?

Reply:
I think @leonid-s-usov can as part of his work.

Reply:
Seems like I haven't yet stumbled over it trying to run the specific tests on my Mac 😅.  I will handle it as needed.